csharp/src/Ice/Incoming.cs

```diff
@@ -585,7 +616,7 @@ private void handleException(Exception exc, bool amd)
 
                 if(_response)
                 {
-                    _os = new Ice.OutputStream(_instance, Ice.Util.currentProtocolEncoding);
+                    Debug.Assert(_os != null);
```

Comment:
I don't see the point of all these Debug.Asserts.

Reply:
They were there to ensure the output stream has been created somewhere else.